documentation/identifiers.md

```diff
@@ -78,65 +88,94 @@ There are many different kinds of organisations that give, receive or benefit fr
+Most organisations have some sort of official registration number that can be used to uniquely identify them and to look up their details from an official registers or public list. 
 
-Organisation Identifiers take the form:
+There are two parts to an organisation identifier:
 
->*Registration Agency Code* (dash) *Identifier*
+* **A list code** that describes the list the identifier is taken from
+* **An identifier** taken from that list.
 
-The Registration Agency Code is taken from the [International Aid Transparency Initiative](http://iatistandard.org/) (IATI) [Organisation Registration Agency Codelist](http://iatistandard.org/codelists/OrganisationRegistrationAgency/) that provides an open, maintained list of reference codes for many Registration Agencies.
+In 360Giving data we ask publishers to use a list code taken from the [org-id list locator](http://org-id.guide/). This provides an open, maintained list of codes for many different lists around with world, giving a way to identify almost any organisation. 
 
-For example, a charity registered in England and Wales with the [Charity Commission](https://www.gov.uk/government/organisations/charity-commission) and a number of **1070468** will use an [IATI Organisation Registration Agency Code](http://iatistandard.org/codelists/OrganisationRegistrationAgency/) of **GB-CHC**. The organisation identifier in your 360Giving data would be:
+```eval_rst
 
->GB-CHC-1070468
+.. admonition:: For example
 
-However, some organisations have more than one such identifier (e.g. where they are both a company and a charity), and others are not registered anywhere, and so you may only have names, or internal IDs, for them recorded on your data.
+  A charity registered in England and Wales with the Charity Commission of England and Wales, with the charity number '1070468' will use a prefix of ``GB-CHC``. 
 
-To give users of 360Giving data the best chance of joining up information about the same organisations across different data files, we follow a simple methodology to create Organisation Identifiers via the following process.
+  This gives an unique organisation identifier of ``GB-CHC-1070468``
 
-Work your way through the following steps. As soon as a step gives you an identifier, you can stop there and use the given identifier.
+```
 
-1. **Registered company number (UK)**
+### Choose the best identifier
 
-    If you have a registered company number for the organisation, use the Registration Agency Code 'GB-COH-' and the registered number (indicating that the number could be looked up at [Companies House](http://www.companieshouse.gov.uk))
+Some organisations have more than one identifier: they might be a charity **and** a company (charitable companies), or a charity **and** an educational establishment. 
```

Comment:
Or several charity numbers (since this is a repeat case)

Reply:
We know that grantmakers generally colect charity numbers where the organisation has one, but they're highly unlikely to collect the company number; but the 360 Standard gives preference to company numbers. Do we need to revisit this? We could create a mapping from company to charity numbers perhaps? I think only Comic relief have used company numbers, and this was a lot of effort for them to add this info before they published.

Reply:
Tooling for this would be something org-id.guide project would be able to work on in future - and some of David Kane's work also helps with - but care needs to be taken to have something that keeps pace with changes. 

Note that with updates to org-id.guide, the preference for company numbers is only a weak preference. I.e.

* If you have them, they are better at uniquely identifying the legal entity, but if you don't have them, charity numbers really are good enough. 

We might just want to start saying 'Charity number, company number or other registered identifier' and recognising that the mapping between them really should be an intermediary job, not something for the publishers.

Reply:
We are the intermediary :-) We need to develop it

Reply:
@morchickit that could be the case, but that's not going to help us get this documentation through (in this draft).

Reply:
As per my earlier comments and Mor's feedback above, we need to use charity numbers  as the first preference going forward. We've had a lot of feedback from publishers that they don't collect company numbers and its a lot of work for them to add this data. Our Board is keen that we address this. We also need organisations to be able to publish several charity numbers if they want to. So for the documentation, please update it to charity number; and going forward we will need to look at updating the standard schema so users can include more than one charity number in cases where they collect it.

Reply:
Please note that the guidance does not ask anyone to **collect a company number**.

It says if you have one - use it. If you don't - use the charity number you have.

Reply:
You're right Tim, in the current guidance a list of org IDs is given (company, charity, education) and encourages people to use whichever ID they have, including using both company and charity number if they've got them. When Katherine started though she was told by ODS that the advice to give publishers was to use company numbers as they're more stable and its good to encourage this, so this is what we've been doing. This means applications like the one we've developed in Salesforce for use the community foundations prioritise company numbers. If we're no longer giving preference to company numbers that's fine, but we need to make sure we're all aware of this decision and that we don't want to give preference to one over another. Do we want to consider asking for charity numbers as the first priority, for example? We can discuss this with Steven on our weekly catch up if easier; as this isn't so much about the written guidance but about the protocol we follow when advising organisations.